### PR TITLE
Issue #456

### LINKED ISSUE #459: charachter issue on iris hause's page
the copyright character is showing up weird so we need to replace it with &copy

go to Iris's Hause's page
scroll down to the bottom
see error 

to be a normal copyright symbol

### PR DESCRIPTION
**Pull request by eli9701**
fixes #459

### CODE REVIEW

**Comment by eli9701** [comment]
fixes-#456

**Comment by gemohandy** [comment]
456 is another pull request, which should be entirely independant of this pull request. Even ignoring the fact that _it's not_, which is a major problem in and of itself (you seem to have made some significant branching mistakes, which makes everything significantly more complicated), you can't really "fix" a pull request. Did you mean to say this fixes #459?
Even if so, there are a _lot_of files which are being changed here which are unrelated to that issue, and which should not be involved in this pull request in any way, shape, or form.

File: www/artists/cfarrero/gallery.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h2>
                 Fiber, too: Unknown material
             </p>
             <h2>
-                These pieces are Charley Farrero collaboration work with <a href="../jjacobs/index.html"></a>June J. Jacobs.
+                These pieces are Charley Farrero collaboration work with <a href="https://saskartists.ca/artists/jjacobs/">June J. Jacobs.</a>
```
Comment (gemohandy) [inline www/artists/cfarrero/gallery.html]: Avoid absolute links to SaskArtists.

Comment (gemohandy) [reply www/artists/cfarrero/gallery.html]: Also, Charley Farrero is not Iris Hause. She is the only person who should be touched if you're dealing with issue #459.

File: www/artists/dhall/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (gemohandy) [inline www/artists/dhall/index.html]: Why are you creating a page for Don Hall in a pull request to fix Iris Hause's page? This is irrelevant and should not be here. At best, it should be in a different pull request; at worst, it should not exist at all.

File: www/artists/dhall/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet">
+    <li class="home">  <a href= "index.html"> Home</a> </li>
```
Comment (gemohandy) [inline www/artists/dhall/index.html]: You can't have list items outside of a list of some sort, either ordered or unordered.

File: www/artists/dhall/photographs.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<li> <a href= "index.html"> Home</a> </li>
```
Comment (gemohandy) [inline www/artists/dhall/photographs.html]: ...This page lacks all the vital elements of an HTML file. No head, no body, _no HTML tag_...

File: www/artists/ihauser/index.html [hunk 5]
```diff
@@ -39,7 +39,7 @@ <h3>
         </div>
         <div id="footer">
             <h6>
-                © Copyright 2013. All images are the property of the artist. Coded 2013 Mykola Kyba. Updated 2015 Bennett Eidsness. Expanded Images 2018 Ryan Robertson
+                &copy Copyright 2013. All images are the property of the artist. Coded 2013 Mykola Kyba. Updated 2015 Bennett Eidsness. Expanded Images 2018 Ryan Robertson
```
Comment (gemohandy) [inline www/artists/ihauser/index.html]: The only actual change that is needed for this pull request. Literally everything else is completely irrelevant, and _shouldn't be here._

File: www/artists/thall/index.html [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (gemohandy) [inline www/artists/thall/index.html]: Even ignoring the obvious fact that this file shouldn't be here because it's irrelevant to this issue, why does Don Hall have files in a folder named thall? This is exactly the same as the index.html in dhall. Why does he get two copies of his page?
